.env.test

```
@@ -3,3 +3,4 @@ NODE_ENV=production
 # Federation
 LOCAL_DOMAIN=cb6e6126.ngrok.io
 LOCAL_HTTPS=true
+WEB_DOMAIN=cb6e6126.ngrok.io
```

```suggestion
WEB_DOMAIN=cb6e6126.ngrok.io
LOCAL_HTTPS=true
```

Small change just to keep the lines with the same values together.